bash_completion

```diff
@@ -2432,6 +2457,8 @@ _xfunc()
     "$@"
 }
 
+#---------------------------------------
```

Comment:
This can likely go if we add per function `@deprecated` markers.

Reply:
2585250f